README.md

```diff
@@ -28,6 +28,15 @@ Questions, bug reporting and issue tracking are provided by GitHub. Please
 report all bugs by creating a new issue with the bug tag. You can ask
 questions by creating a new issue with the question tag.
 
+Contributing
+------------
+
+Contributing to DTK is easy. Just send us a [pull request](https://help.github.com/articles/using-pull-requests/).
```

Comment:
Dynamic analysis tool will show if the new lines of code are not executed and that's the reviewer responsibility to ensure that the tests are presents and appropriate.